modules/core/shared/src/main/scala/lucuma/core/math/dimensional/measure.scala

```diff
@@ -9,6 +9,8 @@ import coulomb._
 import monocle.Focus
 import monocle.Lens
 import shapeless.tag
+import _root_.cats.Show
```

Comment:
I thought we weren't really using `Show` anymore

Reply:
I'll remove it. That `_root_` crept in anyway.